src/client/ocpp/json/JsonChargingStationClient.ts

```diff
@@ -4,7 +4,7 @@ import ChargingStationClient from '../../ocpp/ChargingStationClient';
 import { Command } from '../../../types/ChargingStation';
 import JsonWSConnection from '../../../server/ocpp/json/JsonWSConnection';
 import Logging from '../../../utils/Logging';
-import { MessageType } from '../../../types/WebSocket';
+import { OCPPMessageType } from '../../../types/ocpp/OCPPCommon';
```

Comment:
Again another unnecessary type renaming that generates tons of changes.

Reply:
All OCPP types begin with OCPP string. 
That type is an OCPP type definition that was incorrectly declared in WebSocket types and it has been moved to the OCPP types directly, so I've just make it follow the same naming scheme.